terms.html

```diff
@@ -76,6 +76,10 @@
     <dt><dfn>Expose</dfn></dt>
     <dd>
       <p>Translated to platform-specific <a class="termref" data-lt="accessibility api">accessibility APIs</a> as defined in the <a href="" class="core-mapping">Core Accessibility API Mappings</a>. [[CORE-AAM-1.1]]</p>
+    </dd>
+    <dt><dfn>Focusable</dfn></dt>
+    <dd>
+      <p>An element is focusable if it can be given the input focus by either the user (for example, using mouse or keyboard) or by script.</p>
```

Comment:
By this definition all natively focusable elements will always be exposed to the accessibility tree. Even if you add `tabindex="-1"` and hide it with ARIA hidden, it's still focusable by a script.

```html
<div aria-hidden ="true">
  <a href="#" tabindex="-1">foo</a> <!-- this is focusable -->
  <input tabindex="-1"> <!-- so is this, right? -->
</div>
```

Also, the definition might need to address concepts such as `@inert` and/or `@disabled`